platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -2480,7 +2480,7 @@ namespace platoon_strategic_ihp
         unsigned long tsStart = ros::Time::now().toNSec() / 1000000;
 
         // If vehicle is not rolling then return
-        if (current_speed_ <= STOPPED_SPEED)
+        if (current_speed_ <= config_.minPlatooningSpeed)
```

Comment:
I think this substitution is also needed in mob_req_cb() so it won't react if a request comes in.